src/code-templates/services/order-service/jest.config.js

```diff
@@ -99,7 +99,20 @@ module.exports = {
   // projects: undefined,
 
   // Use this configuration option to add custom reporters to Jest
-  // reporters: undefined,
+  reporters: [
+    "default",
+    [
+      "@jest-performance-reporter/core",
+      {
+        "errorAfterMs": 1000,
+        "warnAfterMs": 500,
+        "logLevel": "warn",
+        "maxItems": 5,
+        "jsonReportPath": "performance-reports/performance-report.json",
```

Comment:
To avoid multiple tests reports folder, I would put ALL the test reports (e.g. including coverage, others) in a single folder "test-reports"

Reply:
Sounds good.  I will unite them